### ISSUE TITLE
As a user, I can view a Tank record 

### ISSUE THREAD

**Issue by benreyn** [rubyforgood/abalone#317]
Acceptance Criteria

- [ ] A new route is added that maps to Tanks#show
- [ ] A new view is added to this route that shows all of the data for a tank
- [ ] A feature test is added that seeds one tank record in the database, loads this page and asserts that content is on the page for that tank

There is some prior art for how we want show pages in our application to look. See Facilities#new and Animals#new. If this task is picked up before #313, Please add to the scope of that issue, or create a new issue for.

- [ ] A link to this page is added to the `Tanks#index` page. 

If this task is picked up after #315, Please create a new ticket to correct the behavior of that form. It should redirect to this page after a new record is created.
